bazel/linux/run_um_kunit_tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+LINUXIMG=$1
+ROOTFS=$2
+HOSTFS=$(dirname $(realpath $3))
+
+export TMPDIR=$(mktemp -d)
```

Comment:
now that I think about it, bazel runs tests in a temporary directory already?

Maybe you can just use ./? or mktemp --tmpdir="."? don't feel strongly either way - feel free to leave as is. However, it is marginally better to use bazel's tempdir, as ... bazel will own cleaning it up, ensuring it's in the right location on eg a remote worker node, etc.

Reply:
It would probably work, but I don't think it's necessary better. I'm already planning on a new iteration for the testing environment, involving rules and specific providers. In that case we can directly create and declare the new file from bazel, so this should be solved.